api/v2/performanceprofile_types.go

```diff
@@ -144,10 +144,10 @@ type Device struct {
 	InterfaceName *string `json:"interfaceName,omitempty"`
 	// Network device vendor ID represnted as a 16 bit Hexmadecimal number.
 	// +optional
-	VendorID *string `json:"vendor,omitempty"`
+	VendorID *string `json:"vendorId,omitempty"`
```

Comment:
should be `vendorId` or `vendorID` (lowercase or uppercase 'D') ?
Here and in all the following instances.
Everything else lgtm

Reply:
Actually I did'nt stumble upon a good guideline for this (especially in our code)  so I decided to go with a Capital separator. 
I'm fine either way

Reply:
> Actually I did'nt stumble upon a good guideline for this (especially in our code) so I decided to go with a Capital separator.
> I'm fine either way

Pretty much the same here. I'd say:
- let's be consistent with our current API - if relevant (can't remember if we have already someIds)
- let's just quickly review what's the latest standard on ocp/k8s, and file an issue should we change the code.

Reply:
looking at some openshift/k8s projects it seems they all comply to using ID/UID in capitals. changing accordingly